packages/leap/lib/src/entities/physical_entity.dart

```diff
@@ -59,13 +57,56 @@ abstract class PhysicalEntity<TGame extends LeapGame> extends PositionedEntity
           ),
         );
 
+  /// Draws a rect over the hitbox when this returns true.
+  bool get debugHitbox => true;
```

Comment:
This should be false be default right?

I should have put the suggestion that I added on the docs here instead haha, but I feel if this were a field, it would be more flexible.

Reply:
I was trying to match Flame Component `debugMode`, which I think is a property, but I'm not sure why. Does it need to be a property to make hot reloads pick up changes?


Also, ya this should default to false, oops

Reply:
Tbh, I am not sure why too hahhaa.

But in my head, if you make this just a field that you can set to true/false, it is easier to make a debug button in your game to sets it on or off with just a click, and that IMO is a better Dev experience than having to change the code, then hotreload/restart.

Reply:
👍 Fixed